packages/process/_process.pony

```diff
@@ -170,7 +170,17 @@ class _ProcessWindows is _Process
   fun tag _make_cmdline(args: Array[String] val): String =>
     var cmdline: String = ""
     for arg in args.values() do
-      cmdline = cmdline + arg + " "
```

Comment:
I think the changes to `process` should be in a separate commit, if it should be included in this PR

Reply:
OK, I'll make a separate PR.